examples/app-python/README.md

```diff
@@ -5,6 +5,7 @@ This is not production-ready code, but it is useful for learning how the platfor
 
 ## Quickstart
 
+# Building sample application
```

Comment:
I do not feel like these seubheadings are necessary, but if you do feel strongly about it that is OK. If you decide to keep them, please update them from single to triple hash marks (`#` -> `###`) as these are equivalent of H3 in markdown embeded within H2.

Reply:
I think, subheadings makes information more modular. Modified headers. kindly let me know if they look OK now.